src/types.ts

```diff
@@ -1,6 +1,9 @@
 // eslint-disable-next-line no-unused-vars
 import Chart from 'chart.js/auto';
-
+export interface AccessibilityOptions {
+  ariaLabel: string;
```

Comment:
I know camel case is more elegant, but we should try and match the [type definitions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L1554) of react to allow us to easily expand to more `AriaAttributes` in the future.